conf/agent.conf.template

```diff
@@ -128,6 +128,25 @@ cygnusagent.sinks.mysql-sink.mysql_password = xxxxxxxxxxxxx
 # how the attributes are stored, either per row either per column (row, column)
 cygnusagent.sinks.mysql-sink.attr_persistence = column
 
+# ============================================
+# OrionSTHSink configuration
+# sink class, must not be changed
+cygnusagent.sinks.sth-sink.type = com.telefonica.iot.cygnus.sinks.OrionSTHSink
+# channel name from where to read notification events
+cygnusagent.sinks.sth-sink.channel = sth-channel
+# FQDN/IP:port where the MongoDB server runs (standalone case) or comma-separated list of FQDN/IP:port pairs where the MongoDB replica set members run
+cygnusagent.sinks.sth-sink.mongo_hosts = x1.y1.z1.w1:port1,x2.y2.z2.w2:port2,...
+# a valid user in the MongoDB server
```

Comment:
`#  a valid user in the MongoDB server (or empty if authentication is not enabled in MongoDB)`

The same change could be also useful in thee configuration part of the mongo sink (both README.md and template files).